src/main/java/edu/whimc/observations/utils/sql/migration/schemas/Schema_4.java

```diff
@@ -1,16 +1,31 @@
 package edu.whimc.observations.utils.sql.migration.schemas;
 
+import edu.whimc.observations.utils.Utils;
 import edu.whimc.observations.utils.sql.migration.SchemaVersion;
+import org.bukkit.ChatColor;
 
-import java.sql.Connection;
-import java.sql.PreparedStatement;
-import java.sql.SQLException;
+import java.sql.*;
 
 public class Schema_4 extends SchemaVersion {
 
     private static final String ADD_CATEGORY =
             "ALTER TABLE whimc_observations ADD COLUMN observation_color_stripped TEXT;";
 
+    /**
+     * Query to update existing observations with an uncolored version
+     */
```

Comment:
```suggestion
    /** Query to update existing observations with an uncolored version. */
```
Simple JavaDocs like these for variables can just be one-liners. However, I'd argue the variable names are self-documenting and don't warrant a JavaDoc